tests/e2e/e2e_test.go

```diff
@@ -563,3 +671,31 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 		})
 	})
 })
+
+func waitForCustomChainsHealthy() string {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	var created bool
+	continueLoop := true
+	for continueLoop {
+		select {
+		case <-ctx.Done():
+			continueLoop = false
+		case <-time.After(5 * time.Second):
+			cctx, ccancel := context.WithTimeout(context.Background(), 15*time.Second)
+			status, err := cli.Status(cctx)
+			ccancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+			created = status.ClusterInfo.CustomChainsHealthy
+			if created {
+				continueLoop = false
+				existingSubnetID := status.ClusterInfo.GetSubnets()[0]
+				gomega.Ω(existingSubnetID).Should(gomega.Not(gomega.BeNil()))
+				cancel()
+				return existingSubnetID
```

Comment:
I will not insist on this change really, but I just want to comment again that this function should only wait for network health, without returning subnet ID. It is used many times only for waiting. Obtaining subnet id should be done in a different ginkgo.By that does not wait and just calls cli.Status and return status.ClusterInfo.GetSubnets()[0].

Reply:
In all cases we have to wait nevertheless first. Being this a test, I don't see a reason which requires this to mandatorily change, so I'd like to keep it.